CONTRIBUTING.md

```diff
@@ -1,75 +1,76 @@
-Contributing to KEDA Helm Charts
-===
+# Contributing to KEDA Helm Charts
 
 Thanks for helping making KEDA better!
 
 ## Shipping a new version
 
 You can easily release a new Helm chart version:
 
-1. Update the version of the Helm chart in `Chart.yaml`
-2. Package the Helm chart 
-   - For KEDA:
+1. Update the version of the Helm chart in `Chart.yaml`.
+1. Package the Helm chart
+    - For KEDA:
+        ```shell
+        $ helm package keda --destination docs
+        Successfully packaged chart and saved it to: docs/keda-2.4.0.tgz
+        ```
+    - For HTTP Addon:
+        ```shell
+        $ helm package http-add-on --destination docs
+        Successfully packaged chart and saved it to: docs/keda-add-ons-http-0.2.0.tgz
+        ```
+    - For KEDA external scaler for Azure Cosmos DB:
+        ```shell
+        $ helm package cosmosdb-scaler --destination docs
+        Successfully packaged chart and saved it to: docs/keda-external-scaler-azure-cosmos-db-0.1.0.tgz
+        ```
+1. Re-index the Helm repo to add our new version:
     ```shell
-    $ helm package keda
-    Successfully packaged chart and saved it to: C:\Code\GitHub\charts\keda-0.1.0.tgz
+    $ helm repo index docs --url https://kedacore.github.io/charts
     ```
-   - For HTTP Addon:
+1. Commit changes:
     ```shell
-    $ helm package http-add-on
-    Successfully packaged chart and saved it to: /home/ecomaz/src/keda/charts/keda-add-ons-http-0.2.0.tgz
+    git add .
+    git commit -sm "Packaged new Helm chart version"
+    git push origin chart-release
     ```
-3. Move the new chart to the docs folder:
-```shell
-$ mv keda-*.tgz docs
-```
```

Comment:
Can you bring this step back please?

Reply:
This is not required once we pass `--destination docs` option to `helm package` command. Also once I get the package renamed to `azure-cosmos-db-add-on`, the move command will need to be changed to `mv *.tgz docs` to include the Cosmos DB scaler package.

Reply:
Nice, thank you.